src/slurm.jl

```diff
@@ -57,12 +57,13 @@ function launch(manager::SlurmManager, params::Dict, instances_arr::Array,
 
         np = manager.np
         jobname = "julia-$(getpid())"
-        srun_cmd = `srun -J $jobname -n $np -o "$(joinpath(job_file_loc, "job%4t.out"))" -D $exehome $(srunargs) $exename $exeflags $(worker_arg())`
+		jobID = String(ENV["SLURM_JOB_ID"])
+        srun_cmd = `srun -J $jobname -n $np -o "$(joinpath(job_file_loc, "job-$jobID-%4t.out"))" -D $exehome $(srunargs) $exename $exeflags $(worker_arg())`
```

Comment:
maybe instead of making the filename more complex we could create a folder `$jobID` and shard the job files that way.

Reply:
Yes, I think your suggestion is better.

More generally, is it necessary to create these files? I have noticed that, for my workflow at least, they only contain the initiation information. Usefull information, including Julia errors, are printed in the main output file.

Maybe instead of printing output it should only print errors with option `-e` instead of `-o`? Although I guess all these files will be generated regardless of an error and the workspace will be littered with empty `.out` even if all processes are initiated successfully.